components/Review/BizReviewPage.tsx

```diff
@@ -164,6 +172,9 @@ function BizReviewPage({ navigation }: BizReviewPageProps) {
                 description={item.description}
                 srcImage={item.srcImage}
```

Comment:
These images aren't rendering for me and are throwing a `source.uri should no be empty` warning